ucsb_ds_capstone_projects_2021/projects/calcofi1/update2.md

```diff
@@ -27,13 +27,13 @@ In our last progress update, we took a brief look at the visual relationship bet
 
 <img src="https://raw.githubusercontent.com/gavintieng/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/calcofi1/update2_exploratory.png" alt="drawing" />
 
-At this point, we wanted to explore the possibility of a more causal relationship by offsetting the years between the datasets. This way, it would give us a better look if, for example, a higher larvae count 2-4 years ago could potentially predict a higher fishery catch later. We performed a basic linear correlation on the offset data, generating the following graph:
+At this point, we wanted to explore the possibility of a more causal relationship. We decided to calculate the correlation of Larva_(t) and Fish_(t+x), with x being between (-2, 7).  This way, it would give us a better look if, for example, a higher larvae count 2-4 years ago could potentially predict a higher fishery catch later. We performed a basic linear correlation on the offset data, generating the following graph:
```

Comment:
Latex can be used in Jupyter book: https://jupyterbook.org/content/math.html
You might find Mathpix helpful: https://mathpix.com